docs/data/data-grid/api-object/api-object.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Data Grid - API object
+---
+
+# Data grid - API object
+
+<p class="description">Interact with the grid using its API.</p>
+
+## What is the API object
+
+The API object is an interface containing the state and all the methods available to programmatically interact with the grid.
+
+You can find the list of all the API methods on the [GridApi page](/x/api/data-grid/grid-api/).
+
+:::warning
+All the methods prefixed by `unstable_` are private.
+We don't recommend using them as they can be removed, renamed, or reworked at any time.
+
```

Comment:
Paragraphs inside callouts have `margin: 0`. I found weird to not have any space between them, so better to keep inside the same paragraph.

```suggestion
```